lib/mail_room/mailbox.rb

```diff
@@ -84,6 +86,7 @@ def deliver(message)
       body = message.attr['RFC822']
       return true unless body
 
+      MailRoom.structured_logger.info("#{context} Delivering #{message.attr['UID']} through #{delivery.class.name} (#{message.attr['RFC822.SIZE']} bytes)")
```

Comment:
Kind off the same like below, I think better input would be:

`uid: message.attr['UID'], deliverer: delivery.class.name, byte_size: #{message.attr['RFC822.SIZE']}.merge(context)`